app/models/lpp/LPPDetails.scala

```diff
@@ -122,3 +125,12 @@ case class LPPDetailsMetadata(
 object LPPDetailsMetadata {
   implicit val format: OFormat[LPPDetailsMetadata] = Json.format[LPPDetailsMetadata]
 }
+
+case class TimeToPay(
+                       TTPStartDate: LocalDate,
+                       TTPEndDate: LocalDate
```

Comment:
I got a bit confused about whether these parameters are optional with in the timeToPay field? Are they optional or mandatory? The spec seems to suggest they are optional?

Reply:
I agree it is not clear, if TPP has been set up I would expect/assume there to be a start date otherwise the list item would be empty, but I guess theoretically there could not be an agreed end date yet, I'll make the end Date optional